packages/feedback-spa/package.json

```diff
@@ -43,7 +43,7 @@
   },
   "devDependencies": {
     "@angular-builders/jest": "9.0.1",
-    "@angular-devkit/build-angular": "0.901.0",
+    "@angular-devkit/build-angular": "^0.803.24",
```

Comment:
Why this is downgraded. any reason.

Reply:
with Angular 8 this is the perfect version we should use.

Reply:
But is the `0.901` version causing any problems?

Also, any specific reason this app is built in Angular 8 and not 9?

Reply:
Yes 0.901 version was causing an error so i removed it.
I chose Angular 8 because its LTS version.

Reply:
Can you please rebase your code and push because your changes will replace the updated version

Reply:
Rebased.

![image](https://user-images.githubusercontent.com/12980398/80405475-4af76300-88e0-11ea-98bb-7e5ebc126e02.png)